platforms/pano_logic_g2.py

```diff
@@ -0,0 +1,183 @@
+# Support for the Pano Logic Zero Client G2
```

Comment:
Please add a bunch of comments to the top of this file on where to find more information.